src/main/java/com.tourguide.tourguide2/TourGuideApplication

```diff
@@ -0,0 +1,13 @@
+package com.tourguide.tourguide2;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TourGuideApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(TourGuideApplication.class, args);
+    }
+
+}
```

Comment:
I added in a TourGuideApplication, or at least the example of one as there was no TourGuideApplication and I do not know what the plan in place for the project is. I was not able to add in a proper Java Class however as there seems to be a bigger issue that is too large for me to do anything about.
Source: [https://martinfowler.com/bliki/TechnicalDebt.html](https://martinfowler.com/bliki/TechnicalDebt.html)

Reply:
What's the bigger issue that you mention?

Is this class needed?  I do see an Application class, though it is a bit buried.  https://github.com/bolandmj/TourGuide/blob/main/complete/src/main/java/com/example/springboot/Application.java

Reply:
Oh then the mistake is on me. I couldn't find the src directory where they would usually be and had just assumed that something had gone wrong with the initial creation of their project that resulted in them not having an Application class. Now I realize it was underneath another directory. Although that application class still seems odd, I wouldn't expect that and the controller to end with .java